impress-org/givewp
# Add option to display company name on donor wall

Resolves #5374

## Description

This PR adds a checkbox option to the Donor Wall block, and support for a new shortcode attribute passed to `[give_donor_wall]` shortcode, that allows site admin to choose whether to display the donor's company name or not. This option is turned OFF by default. 

How to use this newly added option?

**Donor Wall block** 
A new checkbox option named `Show Company Name` is added in the Donor Wall block settings panel. 

**[give_donor_wall] shortcode** 
The shortcode attribute responsible for displaying the company name is `show_company_name`. In order to display the company name, this attribute value must be set to `true`;
Example: `[give_donor_wall show_company_name="true"]` 


## Affects
Donor Wall template. 
Donor Wall block.
`[give_donor_wall]` shortcode.



## Visuals

![image](https://user-images.githubusercontent.com/4222590/95994667-da61e580-0e30-11eb-912b-f70d77aa3d35.png)



## Pre-review Checklist




## Testing Instructions

1. Add a new site page called Donor Wall
2. Add Donor Wall block and try to check/uncheck the `Show Company Name` option multiple times. The company name should appear or disappear depending on current option value.
3. Add Donor Wall using `[give_donor_wall show_company_name="true"]` shortcode, and save the page.
4. Preview the page. The company name should be displayed.
